docs/src/views/FocusTrapZone.tsx

```diff
@@ -0,0 +1,155 @@
+import * as React from 'react'
+import { Header } from '@stardust-ui/react'
+import DocPage from '../components/DocPage'
+import { link, code } from '../utils/helpers'
+
+import CodeSnippet from '../components/CodeSnippet'
+
+export default () => (
+  <DocPage title="Focus Trap Zone">
+    <Header as="h2">Content</Header>
+    <ul>
+      <li>{link('Overview', '/focus-trap-zone#overview')}</li>
+      <li>{link('Usage', '/focus-trap-zone#usage')}</li>
+      <li>
+        {link(
+          'Override FocusTrapZone settings',
+          '/focus-trap-zone#override-focustrapzone-settings',
+        )}
+      </li>
+    </ul>
+    <Header as="h2">Overview</Header>
+    <p>
+      FocusTrapZone is used to trap the focus in any html element. Pressing TAB key will circle
+      focus within the inner focusable elements of the FocusTrapZone.
+    </p>
+    <p>
+      Stardust leverages Focus Trap Zone component which is based on the{' '}
+      {link(
+        'Focus Trap Zone from Office UI Fabric.',
+        'https://developer.microsoft.com/en-us/fabric#/components/focustrapzone',
+      )}
+    </p>
+    <Header as="h2">Usage</Header>
+    <p>
+      In Stardust focus trap is applied through accessibility behavior, as for focus zone. To enable
+      focus trap for component, it is needed, in behavior, to set prop {code('trapFocus')} to{' '}
+      {code('true')} with default settings or set an object with desired values for focus trap zone
+      props. {link('Read more about Accessibility Behaviors.', '/accessibility-behaviors')}
+      Currently, it is used for Popup via {code('popupFocusTrapBehavior')} and Dialog via{' '}
+      {code('dialogBehavior')}.
+    </p>
+    <p>FocusTrapZone's props which can be applied in accessibility behavior:</p>
+    <ul>
+      <li>
+        <b>as</b> - Element type the root element will use. Default is "div".
```

Comment:
not sure if we want to document `as`. maybe mark it as internal? the wrap/embed logic is overriding it as far as I remember

Reply:
You can override "div" with `as` prop for focus trap zone container as it wraps component